proto/osmosis/poolmanager/v1beta1/genesis.proto

```diff
@@ -42,6 +42,10 @@ message GenesisState {
   Params params = 2 [ (gogoproto.nullable) = false ];
   // pool_routes is the container of the mappings from pool id to pool type.
   repeated ModuleRoute pool_routes = 3 [ (gogoproto.nullable) = false ];
+
+  // KVStore state
+  TakerFeesToStakersTracker taker_fees_to_stakers_tracker = 4;
+  TakerFeesToCommunityPoolTracker taker_fees_to_community_pool_tracker = 5;
```

Comment:
Noticed that both of the proto messages contain height_accounting_starts_from, but the two fields always have the same values right?(we're not tracking them distinctly). If som can we abstract the field from the messages and have the 6th field of height_accounting_starts_from, then change these two fields to simply be coins?

Reply:
Agreed! Changed here https://github.com/osmosis-labs/osmosis/pull/6804/commits/b49fc97fd6886d01324cd1a6806e089d96c61442